pkg/didcomm/crypto/legacy/authcrypt/authcrypt_test.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
```

Comment:
Do we have a test where the inputs are generated by the python codebase (and pasted into the test?)

Reply:
Yes, they'll come with decrypt.